sys/congure/quic/congure_quic.c

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (C) 2021 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ *
+ * See [RFC 9002, Appendix B](https://tools.ietf.org/html/rfc9002#appendix-B)
+ * and * parts of [RFC 9002, Appendix A]https://tools.ietf.org/html/rfc9002#appendix-A
```

Comment:
```suggestion
 * and parts of [RFC 9002, Appendix A](https://tools.ietf.org/html/rfc9002#appendix-A)
```

The `*` looks like leftover from reflowing, the rest is fixing the link.

Reply:
Taken and squashed.